experiment/measurer/measure_manager.py

```diff
@@ -70,21 +73,29 @@ def measure_main(experiment_config):
     initialize_logs()
     logger.info('Start measuring.')
 
+    # Create data frame container for segment and function coverage info.
+    experiment_coverage_data_container = (
```

Comment:
Please rename this to `detailed_coverage_data`.

Reply:
Renamed it.